pyanaconda/payload/__init__.py

```diff
@@ -558,6 +560,7 @@ def selectGroup(self, groupid, default=True, optional=False):
 
         self.data.packages.groupList.append(grp)
 
+    # TODO: this function is never used
```

Comment:
IMHO leave it here.

Reply:
Yeah, we definitely should keep complementary functions (eq. add/remove/modify) even if they are not used at the moment. Even though it would be good to have test coverage for them in such case, so that we are reasonably sure the code still works when real code starts using it.

Reply:
Kept.